pack/tcu/bbt_encounter.json

```diff
@@ -0,0 +1,404 @@
+		"clues": 1,
+		"code": "05329",
+		"double_sided": true,
+		"encounter_code": "before_the_black_throne",
+		"faction_code": "mythos",
+		"flavor": "From the horizonless breach in the gaping maw of the sky, you have traveled an impossible distance, and yet you are still many leagues from the nighted throne where Azathoth stirs.",
+		"illustrator": "Reiko Murakami",
+		"name": "The Cosmos Beckons",
+		"pack_code": "bbt",
+		"position": 329,
+		"quantity": 1,
+		"stage": 1,
+		"text": "[action] Spend X clues: Draw the top X cards of the Cosmos. Choose one to put into play using its \"<b>Cosmos</b> -\" instructions, and move to it. Shuffle the rest back into the Cosmos.\n<b>Objective</b> - Only investigators at Hideous Palace may spend the requisite number of clues, as a group, to advance. (<i>Beware - any investigator who is not there will be killed.</i>)",
+		"type_code": "act"
+	},
+	{
+		"back_flavor": "The winged creature lands on a winding walkway of black stone, extending into a ceaseless horizon of pitched void. It takes some prodding and encouragement before your steed continues onward. Even the most dreadful of beasts are terrified of this place, it seems. Yet still, you can hear the mindless chanting of those who dwell closest to the daemon-sultan, the maddening whining of the pipers, the monotonous drum-beats which thrum endlessly in your mind...",
+		"back_name": "Nucleaus of the Universe",
+		"back_text": "Remove Hideous Palace from the game. Shuffle each location in play other than Court of the Great Old Ones into the Cosmos. Each investigator at those locations is left behind and is killed. Each enemy at those locations is shuffled into the top 5 cards of the encounter deck.\nTake the set-aside The Black Throne and the top 3 cards of the Cosmos, and shuffle them so you cannot tell which is which. Then, put them into play along with empty space, as depicted in \"Location Placement for Act 3\" in the Campaign Guide.\nIn player order, each investigator draws the top card of the encounter deck.",
+		"clues": 2,
+		"code": "05330",
+		"double_sided": true,
+		"encounter_code": "before_the_black_throne",
+		"faction_code": "mythos",
+		"flavor": "Your bones shiver violently with the silent, still cold of this empty place. No light penetrates these black halls. No warmth reaches you, no breeze touches your skin. This must be what death feels like, you think morbidly.",
+		"illustrator": "Reiko Murakami",
+		"name": "In Azathoth's Domain",
+		"pack_code": "bbt",
+		"position": 330,
+		"quantity": 1,
+		"stage": 2,
+		"text": "[action] Spend X clues: Draw the top X cards of the Cosmos. Choose one to put into play using its \"<b>Cosmos</b> -\" instructions, and move to it. Shuffle the rest back into the Cosmos.\n<b>Objective</b> - Only investigators at Court of the Great Old Ones may spend the requisite number of clues, as a group, to advance. (<i>Beware - any investigator who is not there will be killed.</i>",
```

Comment:
Please move the italics for Beware to before the (. Please add a closing ) after 'killed.'